### DISCUSSION TITLE
Post Log Issues - Not going into Logbook - Anyone Else?

### DISCUSSION THREAD

**Discussion by AK8CW-Phil** [magicbug/Cloudlog#1839] [Q&A]
Is anyone else experiencing issues with PostQSO not moving into the Logbook? I've historically always populated my logbook via Post QSO mode but most recently, when I click "Save" it never makes it over into my logbook. I'm using Microsoft Edge - Version 108.0.1462.46 (Official build) (64-bit). on my main computer. My last successful Post QSO to logbook entry was December 5th, 2022. My computer has done a few updates over the last week or so. I'm unsure if that is related to the issues I'm having but I'm guessing maybe not as I have tried a second computer using the Chrome browser with the exact same results.  

Thanks,

Phil 
AK8CW 

![Changes Made](https://user-images.githubusercontent.com/93866819/206912406-6dc23dda-6ff8-42aa-aae0-8b8a7139c4de.jpg)
![w4f](https://user-images.githubusercontent.com/93866819/206912414-8abfb781-d29a-42f3-b39a-65419c3997fc.jpg)

**Comment by AndreasK79** [comment]
I just tried in both master and dev. I have no trouble doing post logging.

Do you see any error messages in dev tools (F12) and console?

**Comment by AK8CW-Phil** [comment]
![image](https://user-images.githubusercontent.com/93866819/206917797-82dc2bd8-579b-4bcc-82c6-0dd69ae75d0b.png)

I see this under Console but that seems to be the only thing.

**Reply by AndreasK79** [reply]
Ok, that one is not related to this problem. Not sure what is going on here.

**Comment by magicbug** [comment]
Is the location your adding the QSO to a part of the Logbook you are seeing?

**Reply by AK8CW-Phil** [reply]
It is. I only have one logbook (location) associated with my current setup. 

![image](https://user-images.githubusercontent.com/93866819/206930937-1fef5f24-c7f7-4c24-8fbd-391242ede6a3.png)

**Reply by magicbug** [reply]
Sent you a private email :) sure we can get to the bottom of it

**Reply by AK8CW-Phil** [reply]
Thanks! It's been flawless until now but it could very well be something in conflict on my end. Just not sure where to start after trying it on two computers and different browsers. Thanks!!!!

**Comment by magicbug** [comment]
As per private email to Phil, this should be sorted now was a MySQL config issue.

**Comment by AK8CW-Phil** [comment]
That fixed it, Peter!! Thanks for the prompt response and quick fix!
